DEVELOPERS.md

```diff
@@ -1,7 +1,7 @@
 # Cicero Development Guide
 
 ## ❗ Accord Project Development Guide ❗
-We'd love for you to help develop improvements to Cicero technology! Please refer to the [Accord Project Development guidelines][apdev] we'd like you to follow.
+We'd love for you to help develop improvements to Cicero technology! Please refer to this Accord Project Development guidelines we'd like you to follow.
```

Comment:
Can you also change "Cicero technology" to "Accord Project technology"? And since you are removing the `apdev` link, you can also remove line 34 in this file.